### PR TITLE
ResNet50 || ImageNet Inference || InferenceX Team of Didi Cloud

### PR DESCRIPTION
**Pull request by PengQi220**
Didi Cloud submissions for ImageNet inference with Resnet50 on 1 P4 / 16 GB / 8 vCPU.

### CODE REVIEW

File: ImageNet/inference/DidiCloud_resnet50_1p4_ifx.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    "version": "v1.0",
+    "author": "InferenceX Team of Didi Cloud",
+    "authorEmail": "caijinping@didiglobal.com, yangjintao@didiglobal.com",
+    "framework": "ifx",
+    "codeURL": "https://github.com/didiyun/dawnbench-ifx",
+    "commitHash": "4bb843cc4d195e47fd722dc7f597020154f165fd",
+    "model": "ResNet50",
+    "hardware": "Didi Cloud [1 P4 / 16 GB / 8 vCPU]",
+    "latency": 1.5439,
+    "cost": 3.21e-7,
```
Comment (codyaustun) [inline ImageNet/inference/DidiCloud_resnet50_1p4_ifx.json:11]: I am not familiar with Didi cloud. Please give me a link to pricing information and the instance type so I can verify the cost.

Comment (PengQi220) [reply ImageNet/inference/DidiCloud_resnet50_1p4_ifx.json:11]: > I am not familiar with Didi cloud. Please give me a link to pricing information and the instance type so I can verify the cost.

Thank you for your reply. 
You can get the price information by visiting [Didi cloud gpu details](https://www.didiyun.com/production/gpu.html#charge)，Also you can create an instance by visiting [Didi cloud create an innstance](https://app.didiyun.com/new/#/dc2/add?cpuNum=8&memorySize=17179869184&bandwidth=1024&srvType=ebs.gpu.p4&zoneId=&regionId=).

image

**Comment by FullStackHan** [comment]
@PengQi220 
hi，I checked the github code you submitted, Did the latency you reported not include the time for image pre-processing?

image

**Comment by PengQi220** [comment]
@FullStackHan Yes, Only include Inference time. Consistent with published results.

**Comment by codyaustun** [comment]
@FullStackHan @PengQi220 Yes, as currently defined, we do not require image preprocessing time to be included but welcome it as additional information (see issue #72).

File: ImageNet/inference/DidiCloud_resnet50_1p4_ifx.json (hunk 1 above)
Comment (codyaustun) [reply ImageNet/inference/DidiCloud_resnet50_1p4_ifx.json:11]: Thanks looks good to me!

**Pull request merged**